### PR TITLE
Set metadata for chunks compressed before 2.0

### LINKED ISSUE #2452: Update stats for chunks compressed with previous versions
When we compress chunks we save the statistics from before compression so the planner has meaningful statistics and our row estimate functions also make use of this. Those stats are currently missing from chunks compressed before that change was implemented.

### PR DESCRIPTION
**Pull request by svenklemm**
Fixes #2452

### CODE REVIEW

**Comment by erimatnor** [comment]
@svenklemm Please provide a description of changes and why they needed in the commit message and PR description. This is to aid reviewing and for later understanding these changes in the log.

File: test/sql/updates/post.compression.sql [hunk 1]
```diff
@@ -14,3 +14,18 @@ INNER JOIN _timescaledb_catalog.hypertable hypertable ON (chunk.hypertable_id =
 WHERE hypertable.table_name = 'compress' and chunk.compressed_chunk_id IS NULL;
 
 SELECT * FROM compress ORDER BY time DESC, small_cardinality;
+
+ANALYZE compress;
+select random(),count(*) from compress;
```
Comment (pmwkaa) [inline test/sql/updates/post.compression.sql:19]: What is the idea behind using random here?

**Comment by erimatnor** [comment]
@svenklemm Is this PR obsolete now given https://github.com/timescale/timescaledb/pull/2610? Or, is this only update testing? Since there is no description in the PR it is unclear and title seems to indicate something else. Can you clarify and make this pass tests if it is still to be merged?

**Review by erimatnor — COMMENTED** [review]
Looks like I was requested to re-review this PR, but I am not sure what is going on with this change. The PR title doesn't seem to reflect the changes here and there is no description provided. Also, the referenced issue does not seem fixed by the PR as it only adds a test.

File: sql/updates/latest-dev.sql [hunk 2]
```diff
@@ -63,3 +63,26 @@ CREATE INDEX continuous_aggs_materialization_invalidation_log_idx ON
...
+  FOR plain_chunk IN
+    SELECT
+      *
+    FROM
+      _timescaledb_catalog.chunk comp
+    WHERE
+      compressed_chunk_id IS NOT NULL
+  LOOP
+    SELECT format('%I.%I',schema_name,table_name) INTO comp_chunk FROM _timescaledb_catalog.chunk WHERE id = plain_chunk.compressed_chunk_id;
+    EXECUTE format('SELECT sum(_ts_meta_count) FROM %s', comp_chunk) INTO rowcount;
```
[the remark this answers is not in the document]
Comment (svenklemm) [reply sql/updates/latest-dev.sql]: _ts_meta_count is a column of compressed chunks

File: test/sql/updates/post.compression.sql (hunk 1 above)
Comment (svenklemm) [reply test/sql/updates/post.compression.sql]: this was only temporary to debug the update test

**Review by erimatnor — APPROVED** [review]

**Review by pmwkaa — APPROVED** [review]

**Pull request merged**